Source/Scene/Model/ClassificationModelDrawCommand.js

```diff
@@ -333,7 +335,7 @@ ClassificationModelDrawCommand.prototype.getCommands = function (frameState) {
     this._commandList.push(command);
   }
 
-  return this._commandList;
+  result.push.apply(result, this._commandList);
```

Comment:
What do you mean by "update any calling code accordingly" ? If I did something like this...

`const commands = drawCommand.pushCommands(frameState, frameState.commandList)`

...there isn't any reason for me to use the `commands` var because it's already in the `frameState`'s command list